ledger-service/http-json/src/itlib/scala/http/AbstractHttpServiceIntegrationTest.scala

```diff
@@ -734,6 +740,37 @@ abstract class AbstractHttpServiceIntegrationTest
     } yield succeed
   }
 
+  "get all parties using the legacy token format" in withHttpServiceAndClient {
+    (uri, _, _, client, _) =>
+      import scalaz.std.vector._
+      val partyIds = Vector("P1", "P2", "P3", "P4").map(getUniqueParty(_).unwrap)
+      val partyManagement = client.partyManagementClient
+      val _ =
+        headersWithPartyAuthLegacyFormat(List())
```

Comment:
I think this line is unnecessary

Reply:
Thanks, since it doesn't seem like it does any harm, I'll leave it in so not to block 1.18.1 but I'll do a couple of follow up PRs to remove this from both `release/1.18.x` and `main`.

Reply:
Ok, I'll "backport" this then: https://github.com/digital-asset/daml/pull/12331

Reply:
Addressed by https://github.com/digital-asset/daml/pull/12325/commits/4436b5844db5c345aac9889082defeb47c8d234c